x/instrumentation/test_instrumentation.go

```diff
@@ -1,21 +1,21 @@
 package instrumentation
 
 import (
-	"github.com/tendermint/tendermint/libs/log"
-	"go.uber.org/zap"
 	"os"
 	"strings"
+
+	"github.com/tendermint/tendermint/libs/log"
+	"go.uber.org/zap"
 )
 
 const (
 	// PeggyTestMarker is used in debugging logging statements for logs that are interesting for the Peggy test environment
 	peggyTestMarker = "peggytest"
 	kindMarker      = "kind"
 
-	Startup       = "Startup"
-	EthereumEvent = "EthereumEvent"
-	CosmosEvent   = "CosmosEvent"
-	// TODO: rename, there isnt ethereumprophecyclaim, is there?
+	Startup                        = "Startup"
+	EthereumEvent                  = "EthereumEvent"
+	CosmosEvent                    = "CosmosEvent"
 	EthereumProphecyClaim          = "EthereumProphecyClaim"
```

Comment:
Should resolve this TODO By renaming it to EthereumBridgeClaim, and any relevant changes to the watcher tests.